src/cli/commands/licenses.js

```diff
@@ -49,25 +49,23 @@ async function getManifests(config: Config, flags: Object): Promise<Array<Manife
 
 async function list(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
   const manifests: Array<Manifest> = await getManifests(config, flags);
-
   if (flags.json) {
     const body = [];
-
-    for (const {name, version, license, repository, homepage, author} of manifests) {
+    for (const {name, version, license, licenseText, repository, homepage, author} of manifests) {
       const url = repository ? repository.url : homepage;
       const vendorUrl = homepage || (author && author.url);
       const vendorName = author && author.name;
       body.push([
         name,
         version,
         license || 'Unknown',
+        licenseText || 'Unknown',
         url || 'Unknown',
         vendorUrl || 'Unknown',
         vendorName || 'Unknown',
       ]);
     }
-
-    reporter.table(['Name', 'Version', 'License', 'URL', 'VendorUrl', 'VendorName'], body);
+    reporter.table(['Name', 'Version', 'License', 'Disclaimer', 'URL', 'VendorUrl', 'VendorName'], body);
```

Comment:
`License Text` would be more appropriate here than `Disclaimer`. Disclaimers are typically one component of the license text.